phpunit/class-wp-theme-json-test.php

```diff
@@ -2962,6 +2962,71 @@ public function test_remove_insecure_properties_should_allow_indirect_properties
 		$this->assertSameSetsWithIndex( $expected, $actual );
 	}
 
+	/**
+	 * @covers WP_Theme_JSON_Gutenberg::remove_insecure_properties
+	 */
+	public function test_remove_insecure_properties_should_allow_safe_settings() {
```

Comment:
This asserts that the data goes through. Can we also test that it doesn't if the type is incorrect?